packages/amplication-server/src/core/billing/billing.service.ts

```diff
@@ -208,16 +201,6 @@ export class BillingService {
         successUrl: new URL(cancelUrl, this.clientHost).href,
       },
     });
-    await this.analytics.track({
-      userId,
-      properties: {
-        workspaceId,
-      },
-      event:
-        intentionType === "DOWNGRADE_PLAN"
-          ? EnumEventType.WorkspacePlanDowngradeRequest
-          : EnumEventType.WorkspacePlanUpgradeRequest,
-    });
```

Comment:
I don't think we should remove this event.
It provides us visibility from the server side of the process, so we can understand if there are any blockers after that point

Reply:
Synced with Erez about this suggestion, we've decided to keep these events. Please, see the updated issue.